packages/app/prisma-utils/src/errors/prismaError.ts

```diff
@@ -18,7 +18,14 @@ export const isPrismaClientKnownRequestError = (
   error.code.startsWith('P')
 
 export const isPrismaTransactionClosedError = (error: PrismaClientKnownRequestError): boolean =>
-  error.code === PRISMA_TRANSACTION_ERROR && error.message.includes('Transaction already closed')
+  error.code === PRISMA_TRANSACTION_ERROR &&
+  error.message.toLowerCase().includes('transaction already closed')
+
+export const isPrismaUnableToStartTransactionInTimeError = (
+  error: PrismaClientKnownRequestError,
+): boolean =>
+  error.code === PRISMA_TRANSACTION_ERROR &&
+  error.message.toLowerCase().includes('unable to start a transaction in the given time')
```

Comment:
The PRISMA_TRANSACTION_ERROR is used for different types of issues, I am starting to think that we should retry by default and only contemplate the isPrismaTransactionClosedError case

Reply:
yeah, i think it is the best approach, addressing the change